kernel/GBEngine/kutil.cc

```diff
@@ -9386,7 +9403,8 @@ void enterT(LObject &p, kStrategy strat, int atT)
     }
   }
 
-  if ((strat->tailBin != NULL) && (pNext(p.p) != NULL))
+  // letterplace: if p.shift > 0 then pNext(p.p) is already in the tailBin
+  if ((strat->tailBin != NULL) && (pNext(p.p) != NULL) && !(currRing->isLPring && p.shift > 0))
```

Comment:
Bei den Polynomen in T wird nur das Leitmonom geshiftet. Das passiert erst nachdem für das originale Polynom enterT aufgerufen wird, deswegen zeigt pNext von den geshifteten Polynomen bereits auf die neue Kopie.